cmd/commento/main.go

```diff
@@ -17,6 +17,10 @@ func main() {
 		Die(err)
 	}
 	
+	if err := godotenv.Load(); err != nil {
```

Comment:
I've made changes to make the environment variable loading be like the original ruby library.

So, the user will get a .env.development file in the cmd directory so that if they build and run the binary will work immediately.
The other files that godotenv is supposed to look for (.env.development.local, .env.test, .env etc.) will be .gitignore'd so they aren't accidentally checked in.

If the user wants to override the values in .env.development they can add a .env.development.local which will take precedence over .env.development